data/map systems.txt

```diff
@@ -22077,6 +23326,53 @@ system "Mei Yohn"
 			distance 312
 			period 29.0088
 
+system Meissa
+	pos -208.47222 156.86111
+	government Syndicate
+	link "Al Dhanab"
+	link Alpheratz
+	link Atlas
+	asteroids "small rock" 6 5.082
+	asteroids "medium rock" 8 6.6
+	asteroids "large rock" 13 4.224
+	asteroids "small metal" 9 5.28
+	asteroids "medium metal" 10 6.27
+	asteroids "large metal" 15 3.894
+	minables aluminum 1 7.0420918
+	minables tungsten 12 6.825799
+	minables uranium 1 9.7583272
+	haze _menu/haze-133
+	fleet "Small Core Pirates" 3000
+	fleet "Small Core Merchants" 2000
+	fleet "Large Core Merchants" 4000
```

Comment:
@Quantumshark pushed this suggestion as-was because it was certainly good.

I wonder though if it might be worth, as a bit of a surprise, dropping a rare Marauder spawn in here?

Being a 'safe' system, but also not the direct route system, it would be a good place for a well equipped marauder to ambush someone isolated and make off before anyone showed up to witness it or intervene.

As a kind of backwater that's also directly under the Syndicate's nose, it'd be a good place for marauders to have a secret resupply base for their most clandestine, deep-range expeditions. Especially since other, less competent, pirate elements who might loiter around looking for such things are actively suppressed.